doc/getting-started/apl-nuc.rst

```diff
@@ -120,13 +120,14 @@ partition. Follow these steps:
 
    .. code-block:: none
 
-      # mount /dev/sda1 /mnt
+      $ sudo mount /dev/sda1 /mnt
```

Comment:
This is actually not needed. The ESP is already mounted under `/boot`. Can you remove this step and adjust the path in the steps below?